docs/builds.rst

```diff
@@ -80,11 +80,22 @@ The *Sphinx* and *Mkdocs* builders set the following RTD-specific environment va
    :widths: 15, 10, 30
 
    ``READTHEDOCS``, Whether the build is running inside RTD, ``True``
-   ``READTHEDOCS_VERSION``, The RTD name of the version which is being built, ``latest``
+   ``READTHEDOCS_VERSION``, The RTD slug of the version which is being built, ``latest``
```

Comment:
> I am not exactly sure if I understand it. What would it be the build context? Also not sure what it would mean to have "data without it being a nicer UX"... Sorry for that (it seems that you guys are used to some terms when talking about the project, but for me as an external contributor it is a bit difficult to understand).

This is the build context I'm talking about. We have a [proposed plan](https://docs.readthedocs.io/en/latest/development/design/theme-context.html) to change this, but we're now not 100% sure we want additional info here. What's the goal of providing this data?

Reply:
These are environment variable, not the html sphinx context. This is so users can read these vars and do things like `html_title = f"{project} {READTHEDOCS_VERSION}"` https://github.com/readthedocs/readthedocs.org/pull/8237#issuecomment-856092333 (having some extra logic if the version is external). Having this as env var doesn't break the current html context, and it works for all tools. We can omit this change from this PR if you consider, and leave only the change from the version name on the footer.

Another addition for env vars would be https://github.com/readthedocs/readthedocs.org/issues/8260

Reply:
Thank you very much @stsewd for explaining the need for the env vars.

@ericholscher indeed my main reason for exposing these variables is what @stsewd pointed out. I hope to use the information provided by RTD to compose suitable titles/automatic texts in the sphinx templates. Expanding a bit on the example, these env vars would allow something like the following:
```python
if READTHEDOCS_VERSION_TYPE:
	html_title = f"{project} PR #{READTHEDOCS_VERSION_NAME}"
else:
	html_title = f"{project} {READTHEDOCS_VERSION_NAME}"
```
(the difference between the slug and the verbose name being pointed out in https://github.com/readthedocs/readthedocs.org/pull/8237#discussion_r660676316)

They should not interfere at all with the sphinx build process, but rather help the documentation authors to properly display certain properties in the documentation.

Reply:
We are fine adding the environment variable here as this does not affect the Sphinx's build context and it's done in a generic and useful way.